store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/impl/AggregatorIterator.java

```diff
@@ -57,6 +57,9 @@
     public Value reduce(final Key key, final Iterator<Value> iter) {
         // Get first Value. If this is the only Value then return it straight
         // away;
+        if (!iter.hasNext()) {
+            return null;
```

Comment:
can you re-arrange to have 1 return statement.

Reply:
Still not sure if this fix should even be here. Should `AggregatorIterator.reduce` break or return null when asked to aggregate null? This iterator runs within Accumulo to perform reductions